src/sandbox/mod.rs

```diff
@@ -6,9 +6,7 @@ use anyhow::Result;
 use bitflags::bitflags;
 use derive_builder::Builder;
 use getset::Getters;
-use std::collections::HashMap;
```

Comment:
this was from `make lint` :/ probably should be it's own PR? I'll remove it when a squash as this really has nothing to do with what i'm doing.

Reply:
You can if you'd like to. :+1:

Reply:
I'll get right on it.